yourls/content.md

```diff
@@ -0,0 +1,56 @@
+# What is YOURLS?
+
+YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener. You'll have full control over your data, detailed stats, analytics, plugins, and more. It's free.
+
+> [github.com/YOURLS/YOURLS](https://github.com/YOURLS/YOURLS)
+
+%%LOGO%%
+
+# How to use this image
+
+```console
+$ docker run --name some-%%REPO%% --link some-mysql:mysql -d %%IMAGE%% \
+    -e YOURLS_SITE="https://example.com"
```

Comment:
This `-e` needs to be before `%%IMAGE%%` to work, doesn't it?

Reply:
Oooh, right!